cernopendata/modules/fixtures/data/articles/getting_started/md/alice.md

```diff
@@ -0,0 +1,32 @@
+# Learn how to use the ALICE virtual machine to have a first look at ALICE events and use analysis tools
+
+## "How do I start the ALICE software?"
+
+After successfully installing the ALICE VM you should contextualise it as described here [link to installation guide](../../vms/md/alice.md). Note that you should pick from the CernVM market the "ALICE OpenAccess" item from the list of available ALICE contexts. When starting the VM, this will automatically login the user “alice” (password “alice”) and start a graphical user interface allowing to run the ALICE masterclasses as well as a basic tutorial on how to run a custom analysis on ALICE data. There is no need to setup the software; both the software tools and the environment are automatically set to use the supplied analysis tools.
+
+![](../../../../../theme/static/articles/alice/get_started_1.png)
```

Comment:
I tried quickly locally, and I don't see images, I think the "theme" is extra here.

Can you check browser console? The display is asking for `http://0.0.0.0/theme/static/articles/alice/get_started_1.png` while the correct URL should be  `http://0.0.0.0/static/articles/alice/get_started_1.png`.

So I guess you have extra `/theme/` everywhere...
The article

Reply:
You could probably directly use `/static/...` when referring to images...